content/blog/2020/11/2020-11-03-gsoc-report.adoc

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Google Summer of Code 2020 summary"
+tags:
+- gsoc
+- gsoc2020
+- jenkins
+- community
+- events
+- mentor
+author: oleg_nenashev
+description: This blogpost summarizes Google Summer of Code
+---
+
+With the mentor summit and the project retrospectives finished in October,
+now we can call Google Summer of Code 2020 officially over in the Jenkins community.
+On behalf of the Jenkins org team, we would like to thank all participants: students, mentors, applicants, and dozens of other contributors who participated in the project this year.
+Google Summer of Code would not be successful without the active participation of the Jenkins community.
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+If you follow the Jenkins blog, you may have already seen link:/node/tags/gsoc2020/[many GSoC 2020 articles] created by the project teams.
+Here I would like to focus on the key highlights from the project.
+
+== Projects
+
+In 2020 we had seven students working in the Jenkins mentoring organization.
+We had 6 projects focused on Jenkins and one project focused on Jenkins X.
+As usual, in GSoC we focused on problems important to the Jenkins users and community members.
+The projects delivered highly anticipated new features and key architecture changes
+needed for long-term evolution of Jenkins.
+
+Here are the projects we had this year:
+
+* link:/projects/gsoc/2020/projects/custom-jenkins-distribution-build-service[Custom Jenkins distribution build service] by link:/blog/authors/sladyn98[Sladyn Nunes]
+* link:/projects/gsoc/2020/projects/external-fingerprint-storage[External Fingerprint Storage] by link:/blog/authors/stellargo[Sumit Sarin]
+* link:/projects/gsoc/2020/projects/git-plugin-performance[Git Plugin Performance Improvements] by link:/blog/authors/rishabhbudhouliya[Rishabh Budhouliya]
+* link:/projects/gsoc/2020/projects/github-checks[GitHub Checks API for Jenkins Plugins] by link:/blog/authors/XiongKezhi[Kezhi Xiong]
+* link:/projects/gsoc/2020/projects/jenkins-x-apps-consolidation[Jenkins X: Apps/Addons consolidation] by link:/blog/authors/nodece[Zixuan Liu]
+* link:/projects/gsoc/2020/projects/machine-learning[Machine Learning Plugins for Data Science] by link:/blog/authors/loghijiaha[Loghi Perinpanayagam]
+* link:/projects/gsoc/2020/projects/winsw-yaml-configs[Jenkins Windows Services: YAML Configuration Support] by link:/blog/authors/buddhikac96[Buddhika Chathuranga]
+
+Please refer to the project pages for more information and links to the blogposts and project demos.
```

Comment:
`Please refer to the project pages for more information and links to the blogposts and project demos.` should be `Please refer to the project pages for more information, links to the blog posts, and project demos.